crypto3/libs/blueprint/include/nil/blueprint/blueprint/plonk/assignment.hpp

```diff
@@ -108,6 +108,8 @@ namespace nil {
                 : public crypto3::zk::snark::plonk_assignment_table<BlueprintFieldType> {
 
             using zk_type = crypto3::zk::snark::plonk_assignment_table<BlueprintFieldType>;
+            using zk_desc_type = crypto3::zk::snark::plonk_table_description<BlueprintFieldType>;
```

Comment:
let's avoid using `assignment` for the new bbf components, `plonk_assignment_table` instead